mw-webapp/src/model/businessModel/PlanForNextPeriod.ts

```diff
@@ -1,5 +1,31 @@
 import {TimeUnit} from "src/model/businessModel/time/timeUnit/TimeUnit";
 
+/**
+ * PlanForNextPeriod props
+ */
+interface PlanForNextPeriodProps {
+
+  /**
+   * PlanForNextPeriod's UUID
+   */
+  uuid: string;
+
+  /**
+   * Job that should be done in next period
+   */
+  job: string;
+
+  /**
+   *  Unit of time measurement for {@link estimationTime}
+   */
+  timeUnit: TimeUnit;
+
```

Comment:
it seems to me that it is necessary to remove the line feed between the properties in the interface

Reply:
Added rule about emptz line before block comments. It make code easier to read.
Do you think that we no need this rule?

Reply:
apparently I missed this moment. Then everything is fine)